addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SessionFixationScanRule.java

```diff
@@ -388,6 +389,8 @@ public void scan() {
                                             + "] with param ["
                                             + currentHtmlParameter.getName()
                                             + "] = NULL (possibly somewhere in the redirects)");
+                            log.debug("isPage200 returned: " + was200);
```

Comment:
Remove? It will be always false. (Same comment for following changes.)

Reply:
I figured it was good to be consistent with the logging, since the existing message lists the status code?

Reply:
I can revert it, just want to be sure we're on the same page.

Also for "following changes" did you mean just the logging or the other isPage200s as well? (I assume logging, but again want to assure we're aligned.)

Reply:
Right, that might be confusing, drop the status code instead? Just say  "Got a non-200 response when ...."?

Yeah, it was for the logging.

Reply:
Perfect  😀